CorsixTH/Lua/research_department.lua

```diff
@@ -486,6 +489,23 @@ function ResearchDepartment:improveMachine(machine)
   end
 end
 
+--[[ Updates the dynamic info for each matching machine found in the player's
+hospital rooms.
+!param machine_id The machine ID for determining matches.
+--]]
```

Comment:
I don't think we usually wrap luadoc comments any more (ignoring the already existing ones)... but I could be wrong.

```suggestion
--! Updates the dynamic info for each matching machine found in the player hospital's rooms.
--!param machine_id The machine ID for determining matches.
```

Reply:
The other functions have wrapped text and I see wrapped code as well. Consider it a sperate issue IMO.

Reply:
With long enough explanations we do wrap lines afaik. But doc changes are indeed a different issue.

@lewri LGTM, any other things you want changed? If not seems mergeable to me.

Reply:
Probably squash it? 22 commits seems a bit too much.

Reply:
There's very few changes comparing the original files to the final versions. Would it be easier to cancel this pull request and make a new one?

Reply:
No, don't worry. I do it all the time :upside_down_face: